### PR TITLE
NXP-21099: fix soffice windows command line testing with parameter string.

### PR DESCRIPTION
**Pull request by richardsd**

### CODE REVIEW

**Comment by atchertchian** [comment]
:+1:

File: nuxeo-services/nuxeo-platform-convert/src/main/resources/OSGI-INF/commandline-soffice-contrib.xml [hunk 1]
```diff
@@ -11,8 +11,9 @@
       <parameterString>--headless --norestore --writer --convert-to #{format} #{sourceFilePath} --outdir #{outDirPath}
       </parameterString>
       <testParameterString>--version</testParameterString>
-      <installationDirective>You need to install OpenOffice/LibreOffice and add soffice to the PATH environment
-        variable.
+      <winTestParameterString>--headless --norestore --writer --convert-to #{format} #{sourceFilePath} --outdir #{outDirPath}
```
Comment (efge) [inline nuxeo-services/nuxeo-platform-convert/src/main/resources/OSGI-INF/commandline-soffice-contrib.xml]: The `testParameterString` and `winTestParameterString` don't take any parameters so this is incorrect.

Comment (efge) [inline nuxeo-services/nuxeo-platform-convert/src/main/resources/OSGI-INF/commandline-soffice-contrib.xml]: Use a command-line without `#{..}` stuff at least, and I'm not sure attempting a conversion on non-existing files is the best thing to do.

Comment (richardsd) [inline nuxeo-services/nuxeo-platform-convert/src/main/resources/OSGI-INF/commandline-soffice-contrib.xml]: No, attempting such conversion does not seem the best approach. Just commited a simpler one.

**Comment by efge** [comment]
👍

**Pull request closed without merging**